docs/proposals/README.md

```diff
@@ -0,0 +1,28 @@
+# Enhancement Proposals
+
+An Enhancement Proposal is a way to propose, communicate, and coordinate on new efforts for the `build` project.
+
+It is inspired from our experience with the
+[Kubernetes](https://github.com/kubernetes/enhancements/blob/master/keps/README.md) and
+[OKD](https://github.com/openshift/enhancements/blob/master/guidelines/README.md) enhancement
+processes.
+
+## Quick start
+
+1. Socialize an idea with others. Make sure others think the work is worth doing, and are willing to review design and code changes required.
```

Comment:
[1] Would be good to mention "how to socialize an idea with others"? 
May be add a link to community call, mailing list.

[2] Who are "others"?

Reply:
> Other contributors of the project you mean, right?

Users and contributors too be precise.